napari/layers/labels/labels.py

```diff
@@ -159,7 +162,7 @@ def __init__(
         super().__init__(
             data,
             rgb=False,
-            colormap=colormap,
+            colormap=self._random_colormap,
```

Comment:
Should we figure out what the correct colormap is going to be and pass it to the `Image` layer so we don't have to reset it below if a color_dict is passed? If it makes things too difficult, it's probably a minor optimization and can be skipped.

Reply:
Good point @kevinyamauchi, but I agree with @ziyangczi i think it is fine to overwrite it later